scripts/lib/fs.ts

```diff
@@ -31,7 +29,18 @@ export async function pathExists(path: string) {
 
 /**
  * Deletes all the files in the folder, but preserves subfolders.
+ *
+ * Assumes the folder exists, but it's fine if it's empty.
  */
 export async function rmFilesInFolder(dir: string): Promise<void> {
-  await $`find ${dir}/* -maxdepth 0 -type f | xargs rm -f {}`.quiet();
+  const dirMembers = await fs.readdir(dir);
+  await Promise.all(
+    dirMembers.map(async (member) => {
+      const fp = path.join(dir, member);
+      const stat = await fs.stat(fp);
+      if (stat.isFile()) {
+        await fs.unlink(fp);
+      }
+    }),
+  );
```

Comment:
That's a nice way of looking for the file. If we want to do it more succinct with the find command, I think we could add the -path parameter, something like:
```suggestion
 await $`find ${dir} -maxdepth 1 -type f -path "${dir}/*" | xargs rm -f {}`;
```

Reply:
TIL. Thanks Arnau!